doc: FOTA over BLE doc update (#8966)
nrfconnect/sdk-nrf: doc/nrf/shortcuts.txt

.. |fota_upgrades_def| replace:: You can upgrade the firmware of the device over the air, thus without a wired connection.
   Such an upgrade is called a FOTA (firmware over-the-air) upgrade.

> .. |fota_upgrades_building| replace:: To create a binary file for an application upgrade, make sure that :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` is enabled and build the application as usual.

Reviewer: ```suggestion
.. |fota_upgrades_building| replace:: To create a binary file for an application upgrade, build the application with the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` option enabled.
```
